packages/ocamlformat_support/ocamlformat_support.0.2/opam

```diff
@@ -0,0 +1,12 @@
+opam-version: "1.2"
+maintainer: "OCamlFormat Team <ocamlformat-team@fb.com>"
+authors: "Pierre Weis"
+homepage: "https://github.com/ocaml-ppx/ocamlformat/tree/support"
+bug-reports: "https://github.com/ocaml-ppx/ocamlformat/issues"
+license: "LGPL-2 with OCaml linking exception"
+dev-repo: "https://github.com/ocaml-ppx/ocamlformat.git#support"
```

Comment:
this is not really helpful: `opam pin add ocamlformat_support --dev` will not work because `format_.ml` is not present in that branch. Maybe you want to add a `[make "format_.ml"] {pin}` or something like that in opam's `build` instructions.